.circleci/config.yml

```diff
@@ -21,6 +21,7 @@ jobs:
               - data-cache
 
         # Get latest Anaconda running
+        - run: sudo apt-get install libxss1
```

Comment:
I guess we'll also have to update the docs ... ?

Reply:
Based on the symptoms and related issue, I suspect that this is a container-specific problem that users won't really see. Travis does not have this problem, and the multiple users I know that run `environment.yml` have not hit this problem on their computers.